contrib/connector/common.go

```diff
@@ -22,6 +22,22 @@ import (
 	"strings"
 )
 
+type Mounter interface {
+	GetFSType(device string) (string, error)
+	Format(device string, fsType string) error
+	Mount(device, mountpoint, fsType string, mountFlags []string) error
+	Umount(mountpoint string) error
+	IsMounted(target string) (bool, error)
+	GetHostName() (string, error)
+	GetHostIP() string
+}
+
+type CommonMounter struct{}
+
+func GetCommonMounter() Mounter {
```

Comment:
s/GetCommonMounter/NewMounter

Reply:
`CommonMounter` or `NewMounter`, because sushi mounter is defined as `FakeMounter`

Reply:
Done.